.github/workflows/README.md

```diff
@@ -0,0 +1,65 @@
+
+The `Everything` workflow runs all the other workflows, in order, as a series of steps within a single job. It is useful for your initial deployment, saving you the extra work of running each of the six workflows individually.
+
+This workflow takes two input parameters, one specifying the hub network type and the other specifying an optional list of zero or more subscription ids (or partial ids).
+
+The hub network type input value can be one of:
+
+- HubNetworkWithAzureFirewall
+- HubNetworkWithNVA
+
+The subscription ids input value can be one of:
+
+- Empty, in which case no subscriptions are deployed.
+- A single value consisting of all or part of the subscription id, e.g. `640251f9`.
+- A series of quoted comma-delimited values consisting of values representing all or part of multiple subscription ids, e.g. `"640251f9","49f510ff","aef2d8e7"`.
+
+There are two `Hub Networking` workflows, but you only need to run one of them. The networking workflow you run is based on whether you choose to implement the Azure Firewall or a Network Virtual Appliance (NVA).
+
+In the default implementation, all workflows are run manually. This behavior can be changed by modifying the corresponding YAML workflow definition files. For example, to trigger workflow on a push or pull request to the repository.
+
+These workflow definitions are implemented using modularized PowerShell scripts in the `scripts/deployments`  path. The main entry point for these scripts is `scripts/deployments/RunWorkflows.ps1`.
+
+## Workflow configuration
+
+These workflows use configuration values from the following locations:
+
+- environment related configuration values are stored in the `config/variables` path.
+- logging related configuration values are stored in the `config/logging` path.
+- network related configuration values are stored in the `config/networking` path.
+- subscription related configuration values are stored in the `config/subscriptions` path.
+
+Additional information on configuration files is available here:
+
+- [Environment configuration files](../config/variables/README.md)
+- [Subscription configuration files](../config/subscriptions/README.md)
+
+In addition to the repository-based configuration files, you will also need to create a [GitHub encrypted secret](https://docs.github.com/en/actions/security-guides/encrypted-secrets) named `AZURE_CREDENTIALS`. This secret should contain the JSON output from the `az ad sp create-for-rbac` command you used to create the service principal - ensure you remove any newline characters present in the JSON representation when you create the secret as those characters will break the workflow.
```

Comment:
Include steps for `NVA_USERNAME` and and `NVA_PASSWORD` secrets which are required for Hub + NVA deployment.

Reply:
done